CHANGELOG.md

```diff
@@ -5,11 +5,17 @@ All notable changes to this project will be documented in this file. This librar
 ## [Unreleased]
 - *None*
 
+### Fixed
+- Updated the core library to v3.1.1, including fixes for `polyfill` and `h3SetToMultiPolygon` (#19)
+- Removed Emscripten Node error handling from built library, fixing stacktraces (#18)
+### Added
+- Added generated API documentation to README (#19)
```

Comment:
This was #17

Reply:
Ah, that must be why I was confused about #19. Sigh.